webapp/client/src/components/FormHeader.js

```diff
@@ -1,15 +1,13 @@
 import PropTypes from "prop-types";
-import styled from "styled-components";
-
-const Intro = styled.p`
-  font-size: var(--text-medium);
-`;
+import FormHeaderIntro from "./FormHeaderIntro";
 
 export default function FormHeader({ title, intro, hideIntro }) {
+  const intros = [].concat(intro);
```

Comment:
why do we need to concat an empty with the intro array?

Reply:
Because it could be either one string/element/object or an array of strings/elements/objects. For the FormHeaderIntro we need an array. This will give us an array for both inputs.